src/lib/app-model.tsx

```diff
@@ -34,17 +35,23 @@ export class AppModel {
     formattedOutput = '';
     jsExports?: StylableExports;
     diagnostics: Omit<Diagnostic, 'node' | 'options'>[] = [];
-    onChange? = (): void => undefined;
+    onChange?= (): void => undefined;
     private _onChangeId?: number = undefined;
     constructor() {
-        const searchParams = new URLSearchParams(document.location.hash.slice(1));
-        const state = searchParams.get('state');
-        const { files, selected } = state
-            ? (JSON.parse(JSONUncrush(decodeURIComponent(state))) as URLState)
-            : createSampleData();
-        this.fs.populateDirectorySync('/', files);
-        this.files = files;
-        this.setSelected(selected || Object.keys(files)[0]);
+        try {
+            const searchParams = new URLSearchParams(document.location.hash.slice(1));
```

Comment:
it's better to recover from the error, lets say by using an empty file-system then to bring the app into a non working state. this solution is here find the place that actually explodes and recover. then keep the error for a notification